js/background.js

```diff
@@ -11,6 +11,7 @@ let activePort = chrome.runtime.onConnect.addListener((port) => {
 
    port.onMessage.addListener((msg) => {
        if( msg.action === 'reload') {
+           // initiate extension and iterate through all tabs
```

Comment:
Ten komentarz nic nie wnosi.

Reply:
Ja uwazam ze to srpawi ze przyszly dev bedzie lepiej rozumial funkcje tej metody.